sample.env

```diff
@@ -0,0 +1,23 @@
+# Galaxy UI environment variables. 
+# This file should be copied to `.env` and adjusted as needed. 
+
+# REACT_APP_API_URL defines the URL of the backend server. 
+# This can be set to either your local setup of the backend server or the production server. 
+# Default setting is to the production server.
+REACT_APP_API_URL="https://osm-stats.hotosm.org"
+
+# HOST defines the URL of the frontend on your local setup.
+HOST=127.0.0.1
+
+# Mapbox access key to display the maps (optional)
+# In order to use the default basemap, you’ll need to set a
+# Mapbox token (https://docs.mapbox.com/help/glossary/access-token) from your Mapbox account.
+# Costs apply to usage of Mapbox maps above 50,000 map loads per month; if you’d like to ask for
+# account support to cover those costs, contact Mapbox Community https://mapbox.com/community.
+
+# REACT_APP_MAPBOX_TOKEN=
```

Comment:
You can uncomment this and fill like saying "----Your access token----" , since it is sample env user will fill the key